### PR TITLE
127: Add coordinates to search backend

### PR DESCRIPTION
**Pull request by Schedulaar**
Closees #127

### CODE REVIEW

**Comment by elKei24** [comment]
I am not competent enough in the backend yet for reviewing this, but we should review and merge this quickly, so I can integrate it in #129. I did very similar changes there 😢

**Comment by Schedulaar** [comment]
I looked at your chnages, but afaik, they should be easily resolvable

File: backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/webservice/dataloader/AcceptingStoreDataLoader.kt [hunk 1]
```diff
@@ -14,14 +14,10 @@ val acceptingStoreLoader = DataLoader<Int, AcceptingStore?> { ids ->
         runBlocking {
             transaction {
                 AcceptingStoresRepository.findByIds(ids).map {
-                    AcceptingStore(it.id.value, it.name, it.description, it.contactId.value, it.categoryId.value)
-                }.associateWithKeys<AcceptingStore, Int>({ it.id }, ids)
+                    if (it == null) null
```
Comment (maxammann) [inline backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/webservice/dataloader/AcceptingStoreDataLoader.kt:17]: This is a little bit weird. The accepting store shouldn't be null I think

**Review by maxammann — APPROVED** [review]

File: backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/webservice/dataloader/AcceptingStoreDataLoader.kt (hunk 1 above)
Comment (Schedulaar) [reply backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/webservice/dataloader/AcceptingStoreDataLoader.kt:17]: The user might always query ids for which the accepting store is null

**Review by RoAnBu — APPROVED** [review]

**Pull request merged**